datastructure/test/test_graphlist.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+from datastructure.graphlist import Vertex, Graph
+
+
+class TestGraphlist(unittest.TestCase):
+
+    def test_add_empty(self):
```

Comment:
at minimum tests should check a construction of a graph and see if things are connected that you know should be connected.

Reply:
How to check that?

Reply:
make a graph, add some vertexes, add some edges and verify that the vertexes are connected to the vertexes you expect they should be.

Reply:
I don't quite understand how the code works, how to check if they are connected?

Reply:
go to a vertex (since you have a dictionary of them all go to one by using its lookup value). Then check that its edges point to the nodes they were setup to be connected too, and viceversa.